### PR TITLE
Feature/fetch call functionality

### LINKED ISSUE #10: Fetch Calls

### PR DESCRIPTION
**Pull request by aominhlong**
What's this PR do?
Fetch data

Where should the reviewer start?
scripts.js
apiCalls.js

How should this be manually tested?
Run npm start when in the main directory
Go to http://localhost:8080/

Any background context you want to provide?
N/A

What are the relevant tickets?
resolve #10 

Screenshots (if appropriate)
N/A

Questions:
None

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,4 +1,35 @@
 // Your fetch requests will live here!
+let users = `https://what-s-cookin-starter-kit.herokuapp.com/api/v1/users`
+let ingredients = `https://what-s-cookin-starter-kit.herokuapp.com/api/v1/ingredients`
+let recipes = `https://what-s-cookin-starter-kit.herokuapp.com/api/v1/recipes`
```
Comment (mattruder) [inline src/apiCalls.js:4]: this file looks good!

File: src/scripts.js [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 
-const user = new User(users[getRandomIndex(users)])
-const recipeRepo = new RecipeRepository(recipes);
+let allData = []
+let userData;
+let user;
+let recipeRepo;
+let recipes;
+
+getData().then(data => (allData.push(data)))
```
Comment (nhodnett) [inline src/scripts.js:15]: We might be able to refactor this into a cleaner function?

File: src/scripts.js [hunk 3]
```diff
@@ -88,14 +102,16 @@ function displayNewImages() {
 }
 
 function viewAllRecipes() {
+console.log('recipes', recipes)
```
Comment (nhodnett) [inline src/scripts.js:105]: We should remove these console.logs()

File: src/scripts.js [hunk 4]
```diff
@@ -88,14 +102,16 @@ function displayNewImages() {
...
   toCookButtonArea.classList.add("hidden")
 allRecipeList.innerHTML = ""
 filterFavoritesArea.classList.add("hidden")
 foodImagesSection.classList.add("hidden");
 allRecipesSection.classList.remove("hidden");
 viewAllButton.classList.add("hidden");
-recipeRepo.data.forEach((recipe) => {
+recipeRepo.data.recipeData.forEach((recipe) => {
   allRecipeList.innerHTML += `<li class="recipe-in-list" id=${recipe.id}>${recipe.name}</li>`
 })
```
Comment (mattruder) [inline src/scripts.js:116]: When we refactor, lets really do a good job of organizing our functions

**Review by nhodnett — COMMENTED** [review]
Nice work crew! Man these fetch calls are testing... Good perseverance!

**Pull request merged**